src/app/core/models/common/advanced-settings.model.ts

```
@@ -110,6 +114,41 @@ export class AdvancedSettingsModel {
 
   }
 
+  /**
+   * Returns the top level memo options based on whether expenses are grouped by report
+   * @returns string[]
+   */
+  static getTopLevelMemoOptions(reimbursableGroupedBy?: ExpenseGroupingFieldOption, cccGroupedBy?: ExpenseGroupingFieldOption): string[] {
+    let options = this.getDefaultTopMemoOptions();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)